war/src/main/js/components/dropdowns/hetero-list.js

```diff
@@ -21,6 +21,23 @@ function generateHandles() {
   });
 }
 
+function convertInputsToButtons(e) {
```

Comment:
should there be a deprecation warning logged to get developers to adapt code?

Reply:
@timja would you be OK if this were merged without the deprecation warning or would you strongly prefer that the deprecation warning be added before it is merged?

I'd like to be sure this is included in the next weekly so that it is a strong candidate for a backport to the Nov 15, 2023 LTS baseline.

Reply:
I merged with the assumption that we can add the deprecation message in a later pull request